apps/admin-x-activitypub/src/components/feed/ArticleModal.tsx

```
@@ -717,16 +862,100 @@
+                                                commentCount={item.object.replyCount ?? 0}
+                                                last={true}
+                                                layout='reply'
+                                                object={item.object}
+                                                type='Note'
+                                                onClick={() => {
+                                                    navigateForward(item.id, item.object, item.actor, false);
+                                                }}
+                                                onCommentClick={() => {
+                                                    navigateForward(item.id, item.object, item.actor, true);
+                                                }}
+                                            />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)